### PR TITLE
Make the ajax call native JS. No jQuery dependency.

### PR DESCRIPTION
**Pull request by jorisvm**
I like the plugin. Great work, but I don't think we need jQuery for one simple ajax request. Keeping it lighweight for websites that can run without a jQuery dependency.

### CODE REVIEW

**Comment by vanpariyar** [comment]
Thanks @jorisvm for your contribution.

Can you please share your WordPress profile user name i can mention on the Repo

**Comment by jorisvm** [comment]
@jorisvanmontfort is my WordPress user profile.

**Review by vanpariyar — CHANGES_REQUESTED** [review]
Please use `wp_post_views_ajax_object.post_id`

File: assets/js/ajax.js [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-            nonce: wp_post_views_ajax_object.nonce,
-        },
-        success: function (data) {
+document.addEventListener('DOMContentLoaded', function () {
+    if (false == document.body.classList.contains("archive")) {
+        let url = wp_post_views_ajax_object.ajaxurl; // this is the object instantiated in wp_localize_script function);
+
+        let data = new FormData();
+        data.append('action', 'wppv_counter');
+        data.append('post_id', p_post_views_ajax_object.post_id);
```
Comment (vanpariyar) [inline assets/js/ajax.js:7]: Please use `wp_post_views_ajax_object.post_id`

File: assets/js/ajax.js [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+        data.append('nonce', wp_post_views_ajax_object.nonce);
+        
+        let xhttp = new XMLHttpRequest();
+        xhttp.onload = function() {
+            let res = JSON.parse(this.responseText);
             //Do something with the result from server
```
Comment (vanpariyar) [inline assets/js/ajax.js:13]: Please remove commented code

**Review by vanpariyar — CHANGES_REQUESTED** [review]

File: assets/js/ajax.js (hunk 1 above)
Comment (jorisvm) [reply assets/js/ajax.js:7]: This commit fixed that: https://github.com/vanpariyar/wp-post-views/pull/33/commits/7249d20e1473390694b67d98c3830a3e08a40389

File: assets/js/ajax.js (hunk 2 above)
Comment (jorisvm) [reply assets/js/ajax.js]: Removed al JS comments in: https://github.com/vanpariyar/wp-post-views/commit/01d155ec86771cea20721458cdceaddc91eb957c. Can you pull / merge my forked version?

**Review by vanpariyar — APPROVED** [review]

**Pull request merged**